src/cli/commands/init.js

```diff
@@ -89,12 +91,17 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       question: 'license',
       default: String(config.getOption('init-license')),
     },
+    {
+      key: 'private',
+      question: 'private',
+      default: '',
```

Comment:
I keep `''` because we don't need add line `private: {boolean}` into `package.json` If users don't set value for private when they run: `yarn init`. And one thing, `true` and `false` are a `boolean` value, not a `string`, so I don't wrapper it in `''`.